<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>559</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Patcher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="But_start_pach">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>631</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start patching</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_add_device">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>631</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add device</string>
    </property>
   </widget>
   <widget class="CommandTable" name="commandtableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>399</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>399</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>399</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>1000</width>
      <height>400</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Ip Address</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Device name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>User</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Password</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stage</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>localhost</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>ivan</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="but_load_command">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Load command</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_create_command">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Create command</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_del_device">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>631</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete device</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>869</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguafe">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionRussian"/>
    <addaction name="actionEnglish"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSave_sattings"/>
   </widget>
   <addaction name="menuLanguafe"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRussian">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionSave_sattings">
   <property name="text">
    <string>Save sattings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommandTable</class>
   <extends>QTableWidget</extends>
   <header>commandtable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
